function-invokers/java-function-invoker/src/test/java/io/sk8s/invoker/java/server/JavaFunctionInvokerApplicationTests.java

```diff
@@ -16,25 +16,44 @@
 
 package io.sk8s.invoker.java.server;
 
+import java.net.URI;
+
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.RequestEntity;
+import org.springframework.http.ResponseEntity;
 import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.context.junit4.SpringRunner;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 /**
  * @author Mark Fisher
  */
 @RunWith(SpringRunner.class)
-@SpringBootTest
-@TestPropertySource(properties = "function.uri=file:target/test-classes" +
-		"?io.sk8s.invoker.java.function.FunctionConfigurationTests.Doubler")
-
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@TestPropertySource(properties = "function.uri=file:target/test-classes"
+		+ "?io.sk8s.invoker.java.function.FunctionConfigurationTests.Doubler")
+@AutoConfigureMockMvc
```

Comment:
Any reason why this test has this annotation, while the other doesn't?

Reply:
No. It's a copy-paste error.